pytorch_lightning/logging/sacred.py

```diff
@@ -0,0 +1,65 @@
+from logging import getLogger
+from time import time
+
+try:
+    import sacred
+except ImportError:
+    raise ImportError('Missing sacred package.')
```

Comment:
pls extend the message, follow the template of the other loggers

Reply:
I am confused, this looks like in the other loggers, e.g. https://github.com/williamFalcon/pytorch-lightning/blob/master/pytorch_lightning/logging/test_tube.py#L35

Reply:
ok, right, but in the logging init is the longer message...

Reply:
I am not sure what I am looking for - could you point me to it? Thanks!

Reply:
ok, I am quite sure that we did it, but maybe it is still pending in other PR

Reply:
Ah, you mean this? https://github.com/williamFalcon/pytorch-lightning/pull/648/files#diff-9fabe2a32ac055ecbb7e3a2fcab65046R46

Reply:
yes =)